spring-boot-admin-server-ui/src/main/frontend/views/applications/notification-filter-settings.vue

```diff
@@ -19,7 +19,7 @@
     <template v-if="!activeFilter">
       <div class="field">
         <p class="control has-inline-text">
-          Suppress notifications on <code v-text="object.id || object.name" /> for
+          <span v-html="$t('applications.suppress_notifications_on', {name: object.id || object.name})" />&nbsp;
```

Comment:
Why it is sometimes `v-t="..."` `v-text="$t(...)"` or `v-html="$t(...)" ... would have expected a consistent way of translating

Reply:
I have used `v-html` wherever the given text contained HTML. Since it is not safe to use `v-html` I tried to stay away from that wherever possible.